File: compiled_lang/java/src/e10.branch.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class E10Branch {
    public static void main(String[] args) throws IOException {
        BufferedReader stdin = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Would you like a toast? [Yes/No]: ");
        String response = stdin.readLine();

        // Java's ternary operator
        String message = response.equals("Yes") ? "That's great!" : "How about a muffin?";

        System.out.println(message);
    }
}

File: compiled_lang/cpp/src/e10.branch.cpp
#include <iostream>
#include <string>

int main() {
    std::cout << "Would you like a toast? [Yes/No]: ";
    std::string response;
    std::getline(std::cin, response);

    // C++'s ternary operator
    std::string message = (response == "Yes") ? "That's great!" : "How about a muffin?";

    std::cout << message << std::endl;

    return 0;
}
